src/scss/components/_dropdown.scss

```diff
@@ -42,10 +42,12 @@ $dropdown-disabled-opacity: 0.5 !default;
             display: none;
         }
     }
-    &.is-hoverable {
-        &:hover {
-            .dropdown-menu {
-                display: inherit;
+    @media (hover: hover) {
```

Comment:
@wesdevpro Interesting. It might be due to 2-in-1. I conditionally applied the hover reaction to devices where the primary input method supports hover; e.g., a mouse.

Reply:
@wesdevpro Let's what happens if I remove the condition.

Reply:
Okay👍🏼Let me know when you push your changes

Reply:
@wesdevpro Please try it out!